the-blue-alliance-ios/View Elements/Events/EventCellViewModel.swift

```diff
@@ -3,13 +3,25 @@ import Foundation
 struct EventCellViewModel {
 
     let eventShortname: String
+    let eventWeek: String?
     let eventLocation: String?
     let eventDate: String?
 
     init(event: Event) {
         eventShortname = event.safeShortName
         eventLocation = event.locationString
         eventDate = event.dateString()
+        
+        /**
+         Only show the week number if the event if it is in Week 1..7
+         Other events (Offseason, Championships, Preseason) already have their descriptions in the table view.
+         */
+        if event.weekString.contains("Week") {
```

Comment:
I don't love this - we could split out the week string calculation section of `weekString` (maybe we call that method `weekString`, and call the existing `weekString` method something else) and call that directly in this case.

Reply:
Will rename the current `weekString` to `title` and make a new `weekString`